qemu/tests/sgx_multi_vms.py

```diff
@@ -0,0 +1,47 @@
+from virttest import error_context
+from virttest import env_process
+from virttest.utils_misc import verify_dmesg
+
+from provider.sgx import SGXHostCapability
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx multi vms boot test:
+    1. Check host sgx capability
+    2. Boot multi sgx VMs
+    3. Verify each VM sgx enabled in guest
+    4. Check each VM sgx qmp cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Boot four guests", test.log.info)
+    timeout = float(params.get("login_timeout", 240))
+
+    sgx_cap = SGXHostCapability(test, params)
+    sgx_cap.validate_sgx_cap()
+    host_total_epc_size = sgx_cap.host_epc_size
+    if params.get("monitor_expect_nodes"):
+        sgx_cap.validate_numa_node_count()
+
+    params['start_vm'] = 'yes'
+    vms = params.get("vms").split()
```

Comment:
vms = params.objects("vms") is also OK

Reply:
hi @zixi-chen , params.objects('vms') also return a list of 'vms values', i.e. ['vm1', 'vm2', 'vm3', 'vm4], in fact, this function will do the split first, then remove the dup items from the list

Reply:
Thanks Xueqiang and Zhenchao, You are right.